tests/spec/core/markdown-spec.js

```diff
@@ -41,69 +45,86 @@ describe("Core - Markdown", function() {
     };
     ops.config.format = "markdown";
     makeRSDoc(ops, function(doc) {
-      expect($(".note em", doc).length).toEqual(1);
-      expect($(".issue em", doc).length).toEqual(1);
-      expect($(".req em", doc).length).toEqual(1);
-      expect($(".req h3", doc).length).toEqual(1);
+      expect(doc.querySelector(".note em")).toBeTruthy();
+      expect(doc.querySelector(".issue em")).toBeTruthy();
+      expect(doc.querySelector(".req em")).toBeTruthy();
+      expect(doc.querySelector(".req h3")).toBeTruthy();
     }).then(done);
   });
 
   it("should remove left padding before processing markdown content", function(done) {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\n    Foo\n    ===\n",
+       "\n\n  Foo\n  ===\n    * list item 1\n    * list item 2\n      * nested list item",
     };
     ops.config.format = "markdown";
     makeRSDoc(ops, function(doc) {
-      expect($("code", doc).length).toEqual(0);
+      expect(doc.querySelector("code")).toBeFalsy();
+      expect(doc.querySelector("#foo").textContent === "Foo");
+      var listItems = doc.querySelectorAll("section > ul:not([class=toc]) > li");
+      expect(listItems.length).toEqual(2);
+      expect(listItems[0].textContent).toEqual("list item 1");
+      var nestedLi = doc.querySelector("li > ul > li");
+      expect(nestedLi).toBeTruthy();
+      expect(nestedLi.textContent).toEqual("nested list item");
     }).then(done);
   });
 
   it("should structure content in nested sections with appropriate titles", function(done) {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\nFoo\n===\n\nBar\n---\n\nBaz\n---\n\n### Foobar ###\n\n#### Foobaz ####\n\nZing\n---\n\n",
+       "\n\nFoo\n===\n\nBar\n---\n\nBaz\n---\n\n### Foobar ###\n\n#### Foobaz ####\n\nZing\n---\n\n",
```

Comment:
As before. Maybe it's a bug in marked.js?, but the output is not what you would expect. You can try it directly in the marked() parser independently of respec. 

> On 5 May 2016, at 8:07 PM, Tobie Langel <notifications@github.com> wrote:
> 
> In tests/spec/core/markdown-spec.js:
> 
> >      }).then(done);
> >    });
> >  
> >    it("should structure content in nested sections with appropriate titles", function(done) {
> >      var ops = {
> >        config: makeBasicConfig(),
> >        body: makeDefaultBody() +
> > -       "\nFoo\n===\n\nBar\n---\n\nBaz\n---\n\n### Foobar ###\n\n#### Foobaz ####\n\nZing\n---\n\n",
> > +       "\n\nFoo\n===\n\nBar\n---\n\nBaz\n---\n\n### Foobar ###\n\n#### Foobaz ####\n\nZing\n---\n\n",
> Same here, why add this extra \n?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>